src/server/routes/nyplApiClient/index.js

```diff
@@ -1,27 +1,35 @@
 import NyplApiClient from '@nypl/nypl-data-api-client';
-import config from '../../../../appConfig.js';
 import aws from 'aws-sdk';
 
+import config from '../../../../appConfig.js';
+import logger from '../../../../logger.js';
+
 const appEnvironment = process.env.APP_ENV || 'production';
+const kmsEnvironment = process.env.KMS_ENV || 'encrypted';
 const apiBase = config.api[appEnvironment];
-const kms = new aws.KMS({
-  region: 'us-east-1',
-});
+let decryptKMS;
+let kms;
 
-function decryptKMS(key) {
-  const params = {
-    CiphertextBlob: new Buffer(key, 'base64'),
-  };
+if (kmsEnvironment === 'encrypted') {
```

Comment:
I feel the structure of script here can be rearranged.

Here's you have a condition to tell the value of `kmsEnvironment ` and yet in `function client(){}` you have another. 

Also in `if (kmsEnvironment !== 'encrypted') {}`, you have a quite independent block of scripts inside, and it returns in the end to end the scripts. I think this block can be moved to the front and leave all the script based on the other condition, `if (kmsEnvironment === 'encrypted') {}`, behind.